### ISSUE TITLE
Admin is broken

### ISSUE THREAD

**Issue by louismunro** [inverse-inc/packetfence#2130]
The admin (and configurator) is broken in the latest devel build.

Catalyst error follows:

```
Couldn't render template "admin/index.tt: file error - admin/index.tt: not found"

Stack Trace

Package	Line	File
Request

bless({
  _log                 => bless({ abort => 0, autoflush => 0, watch_delay => 300 }, "Log::Log4perl::Catalyst"),
  _path                => "admin",
  _read_length         => 0,
  _read_position       => 0,
  _use_hash_multivalue => 0,
  action               => "admin",
  address              => "172.20.20.161",
  arguments            => [],
  base                 => bless(do{\(my $o = "https://172.20.20.98:1443/")}, "URI::https"),
  body_parameters      => {},
  captures             => [],
  cookies              => {
                            CGISESSION => bless({
                              name  => "CGISESSION",
                              path  => "/",
                              value => ["422bdf5361eff460a491ae4220f5c236d46febec"],
                            }, "CGI::Simple::Cookie"),
                          },
  data_handlers        => {
                            "application/json" => sub { ... },
                            "application/x-www-form-urlencoded" => sub { ... },
                          },
  env                  => {
                            "CONTEXT_DOCUMENT_ROOT"          => "/usr/local/pf/html/pfappserver/lib",
                            "CONTEXT_PREFIX"                 => "",
                            "DOCUMENT_ROOT"                  => "/usr/local/pf/html/pfappserver/lib",
                            "GATEWAY_INTERFACE"              => "CGI/1.1",
                            "HTTP_ACCEPT"                    => "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
                            "HTTP_ACCEPT_ENCODING"           => "gzip, deflate, sdch, br",
                            "HTTP_ACCEPT_LANGUAGE"           => "en-US,en;q=0.8,fr-CA;q=0.6,fr;q=0.4,fr-FR;q=0.2",
                            "HTTP_CONNECTION"                => "keep-alive",
                            "HTTP_COOKIE"                    => "CGISESSION=422bdf5361eff460a491ae4220f5c236d46febec",
                            "HTTP_DNT"                       => 1,
                            "HTTP_HOST"                      => "172.20.20.98:1443",
                            "HTTP_UPGRADE_INSECURE_REQUESTS" => 1,
                            "HTTP_USER_AGENT"                => "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36",
                            "HTTPS"                          => "on",
                            "MOD_PERL"                       => "mod_perl/2.0.9",
                            "MOD_PERL_API_VERSION"           => 2,
                            "PATH"                           => "/sbin:/bin:/usr/bin:/usr/sbin",
                            "PATH_INFO"                      => "/admin",
                            "plack.original_request_method"  => "GET",
                            "psgi.errors"                    => *main::STDERR,
                            "psgi.input"                     => bless(do{\(my $o = 140037003315408)}, "Apache2::RequestRec"),
                            "psgi.multiprocess"              => 1,
                            "psgi.multithread"               => "",
                            "psgi.nonblocking"               => "",
                            "psgi.run_once"                  => "",
                            "psgi.streaming"                 => 1,
                            "psgi.url_scheme"                => "https",
                            "psgi.version"                   => [1, 1],
                            "psgix.cleanup"                  => 1,
                            "psgix.cleanup.handlers"         => [],
                            "psgix.harakiri"                 => 1,
                            "QUERY_STRING"                   => "",
                            "REMOTE_ADDR"                    => "172.20.20.161",
                            "REMOTE_PORT"                    => 58079,
                            "REQUEST_METHOD"                 => "GET",
                            "REQUEST_SCHEME"                 => "https",
                            "REQUEST_URI"                    => "/admin",
                            "SCRIPT_FILENAME"                => "/usr/local/pf/html/pfappserver/lib/admin",
                            "SCRIPT_NAME"                    => "",
                            "SERVER_ADDR"                    => "172.20.20.98",
                            "SERVER_ADMIN"                   => "root\@packetfence.packetfence.org",
                            "SERVER_NAME"                    => "172.20.20.98",
                            "SERVER_PORT"                    => 1443,
                            "SERVER_PROTOCOL"                => "HTTP/1.1",
                            "SERVER_SIGNATURE"               => "",
                            "SERVER_SOFTWARE"                => "Apache",
                          },
  headers              => bless({
                            "::std_case"                => {
                                                             "cookie" => "COOKIE",
                                                             "dnt" => "DNT",
                                                             "https" => "HTTPS",
                                                             "upgrade-insecure-requests" => "UPGRADE-INSECURE-REQUESTS",
                                                           },
                            "accept"                    => "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
                            "accept-encoding"           => "gzip, deflate, sdch, br",
                            "accept-language"           => "en-US,en;q=0.8,fr-CA;q=0.6,fr;q=0.4,fr-FR;q=0.2",
                            "connection"                => "keep-alive",
                            "cookie"                    => "CGISESSION=422bdf5361eff460a491ae4220f5c236d46febec",
                            "dnt"                       => 1,
                            "host"                      => "172.20.20.98:1443",
                            "https"                     => "on",
                            "upgrade-insecure-requests" => 1,
                            "user-agent"                => "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36",
                          }, "HTTP::Headers"),
  match                => "admin",
  method               => "GET",
  parameters           => {},
  protocol             => "HTTP/1.1",
  query_keywords       => "",
  query_parameters     => {},
  remote_user          => undef,
  secure               => 1,
  uploads              => {},
  uri                  => bless(do{\(my $o = "https://172.20.20.98:1443/admin")}, "URI::https"),
}, "Catalyst::Request")
Response

bless({
  _log => bless({ abort => 0, autoflush => 0, watch_delay => 300 }, "Log::Log4perl::Catalyst"),
  _response_cb => sub { ... },
  body => undef,
  cookies => {},
  finalized_headers => 0,
  headers => bless({
    "::std_case"   => { "x-catalyst" => "X-Catalyst" },
    "content-type" => "text/html; charset=utf-8",
    "x-catalyst"   => 5.90065,
  }, "HTTP::Headers"),
  status => 500,
}, "Catalyst::Response")
Stash

{
  current_view => "Admin",
  status_msg   => "An error condition has occured. See server side logs for details.",
}
Config

do {
  my $a = {
    "Action::RenderView"                          => {
                                                       ignore_classes => [
                                                                           "DBIx::Class::ResultSource::Table",
                                                                           "DBIx::Class::ResultSourceHandle",
                                                                           "DateTime",
                                                                         ],
                                                       scrubber_func  => sub { ... },
                                                     },
    "default_view"                                => "HTML",
    "disable_component_resolution_regex_fallback" => 1,
    "home"                                        => "/usr/local/pf/html/pfappserver",
    "name"                                        => "pfappserver",
    "Plugin::Authentication"                      => {
                                                       default_realm => "admin",
                                                       realms => {
                                                         admin => {
                                                           credential => {
                                                             class => "Password",
                                                             password_field => "password",
                                                             password_hash_type => "SHA-1",
                                                             password_type => "self_check",
                                                           },
                                                           store => {
                                                             class => "+pfappserver::Authentication::Store::PacketFence",
                                                             user_class => "pfappserver::Authentication::Store::PacketFence::User",
                                                             user_field => "username",
                                                           },
                                                           use_session => 1,
                                                         },
                                                         configurator => {
                                                           credential => {
                                                             class => "Password",
                                                             password_field => "password",
                                                             password_hash_type => "SHA-1",
                                                             password_type => "none",
                                                           },
                                                           store => {
                                                             class => "Minimal",
                                                             users => { _PF_CONFIGURATOR => { roles => ["ALL"] } },
                                                           },
                                                           use_session => 1,
                                                         },
                                                         proxy => {
                                                           credential => { class => "+pfappserver::Authentication::Credential::Proxy" },
                                                           store => {
                                                             class => "+pfappserver::Authentication::Store::PacketFence",
                                                             user_class => "pfappserver::Authentication::Store::PacketFence::User",
                                                             user_field => "username",
                                                           },
                                                           use_session => 1,
                                                         },
                                                       },
                                                       use_session => 1,
                                                     },
    "Plugin::ConfigLoader"                        => {},
    "Plugin::I18N"                                => { installed_languages => { en => "English", fr => "French" } },
    "Plugin::Session"                             => {
                                                       chi_args          => { namespace => "httpd.admin" },
                                                       chi_class         => "pf::CHI",
                                                       cookie_name       => "pfappserver_session",
                                                       expires           => 86400,
                                                       expiry_threshold  => 0,
                                                       verify_address    => 0,
                                                       verify_user_agent => 0,
                                                     },
    "Plugin::Static::Simple"                      => {
                                                       debug => 1,
                                                       dirs => [],
                                                       ignore_dirs => [
                                                         "captive-portal",
                                                         "admin",
                                                         "pfappserver",
                                                         "templates",
                                                         "violations",
                                                       ],
                                                       ignore_extensions => ["cgi", "php", "inc", "tt", "html", "xml"],
                                                       include_path => [
                                                         bless({
                                                           dirs => ["", "usr", "local", "pf", "html", "pfappserver", "root"],
                                                           file_spec_class => undef,
                                                           volume => "",
                                                         }, "Path::Class::Dir"),
                                                         "/usr/local/pf/html/captive-portal",
                                                         "/usr/local/pf/html",
                                                       ],
                                                       mime_types => { woff => "font/woff" },
                                                       mime_types_obj => bless({}, "MIME::Types"),
                                                       no_logs => 1,
                                                     },
    "root"                                        => 'fix',
    "setup_components"                            => {},
    "stacktrace"                                  => { context => 3, verbose => 0 },
    "static"                                      => 'fix',
    "use_request_uri_for_path"                    => 1,
    "View::Admin"                                 => {
                                                       INCLUDE_PATH => [
                                                         bless({
                                                           dirs => ["", "usr", "local", "pf", "html", "pfappserver", "root-custom"],
                                                           file_spec_class => undef,
                                                           volume => "",
                                                         }, "Path::Class::Dir"),
                                                         bless({
                                                           dirs => ["", "usr", "local", "pf", "html", "pfappserver", "root"],
                                                           file_spec_class => undef,
                                                           volume => "",
                                                         }, "Path::Class::Dir"),
                                                       ],
                                                     },
    "View::HTML"                                  => {
                                                       INCLUDE_PATH => [
                                                         bless({
                                                           dirs => ["", "usr", "local", "pf", "html", "pfappserver", "root-custom"],
                                                           file_spec_class => undef,
                                                           volume => "",
                                                         }, "Path::Class::Dir"),
                                                         bless({
                                                           dirs => ["", "usr", "local", "pf", "html", "pfappserver", "root"],
                                                           file_spec_class => undef,
                                                           volume => "",
                                                         }, "Path::Class::Dir"),
                                                       ],
                                                     },
    "View::JSON"                                  => {
                                                       expose_stash => [
                                                         "status",
                                                         "status_msg",
                                                         "error",
                                                         "interfaces",
                                                         "networks",
                                                         "switches",
                                                         "config",
                                                         "services",
                                                         "success",
                                                         "items",
                                                       ],
                                                     },
  };
  $a->{"root"} = $a->{"Plugin::Static::Simple"}{include_path}[0];
  $a->{"static"} = $a->{"Plugin::Static::Simple"};
  $a;
}
pfappserver on Catalyst 5.90065
```

**Comment by jrouzierinverse** [comment]
Did this happen after running the configurator?

James Rouzier
jrouzier@inverse.ca :: +1.514.447.4918 (x115)  ::  http://www.inverse.ca
Inverse inc. :: Leaders behind SOGo (http://www.sogo.nu) and PacketFence (http://www.packetfence.org)

**Comment by jrouzierinverse** [comment]
Fixed by b3bf2909ab0cb8dd7ca449efad0a01ccea6a19e9
